requirements.txt

```diff
@@ -19,6 +19,7 @@ gwdatafind>=1.1.3
 
 # Requirements for full pegasus env
 pegasus-wms.api >= 5.0.3
+GitPython
```

Comment:
Why is this needed? The pegasus API does not require it, and we are not trying to install the full pegasus toolkit (ie `pegasus-plan` and friends) here.

Reply:
I know that this is not needed on CIT but somehow while generating the workflow, I have been asked to install the GitPython package on lho,  llo and hawk when I  built my know env:
```
virtualenv -p /cvmfs/oasis.opensciencegrid.org/ligo/sw/conda/envs/igwn-py39/bin/python env

source env/bin/activate

cd env

git clone https://github.com/gwastro/pycbc.git

cd pycbc
pip install -r requirements.txt
pip install -r companion.txt
pip install .

pip install -r requirements-igwn.txt
pip install .[igwn]
```

Reply:
In any case, while I would like to understand this, let's not include it in this PR as it's not related to the change being proposed.

Reply:
Without wanting to derail the PR (@bhooshan-gadre could you add an issue?) I've seen this when attempting to run workflows at CIT as well